### DISCUSSION TITLE
Search specifies US location, search results outside US in unstructured search

### DISCUSSION THREAD

**Discussion by matkoniecz** [osm-search/Nominatim#3526] [Q&A] [answered]
## What did you search for?

https://nominatim.openstreetmap.org/ui/search.html?q=US%2C+3136+Cherry+St%2C+Toledo

## What result did you get?

https://nominatim.openstreetmap.org/ui/details.html?osmtype=W&osmid=186563067&class=highway

## What result did you expect?

No results or location in US.

This place appears to not exist or be not mapped yet.

**When the result missing completely:**

## Further details

https://nominatim.openstreetmap.org/ui/details.html?osmtype=W&osmid=186563067&class=highway does not seem to list any US place.

Maybe I should have use structured Nominatim query?

https://nominatim.openstreetmap.org/ui/search.html?street=3136+Cherry+St&city=Toledo&country=US seems to work?

**Comment by lonvia** [comment, marked as answer]
The order of the query is so unusual that Nominatim misinterprets it. Use https://nominatim.openstreetmap.org/ui/search.html?q=3136+Cherry+St%2C+Toledo%2C+US and all works well.

Just to clarify: left-to-right addressing like in your example needs the country on the left side. Right-to-left addressing (as usually used in the US) needs the country on the left.
